addons/website_slides/tests/test_ui_wslides.py

```diff
@@ -274,6 +274,7 @@ def test_course_publisher_elearning_manager(self):
 
 @tests.common.tagged('external', 'post_install', '-standard', '-at_install')
 class TestPortalComposer(TestUICommon):
+    # TODO: Check if it is still required after merging portal chatter.
```

Comment:
Note: 
In `/portal/attachment/add` the access token is returned for all users since odoo#159303. It's not what is done via `/mail/attachment/upload` and it seems to work fine. I think this test could be deleted after merging odoo#138233.

Reply:
To be investigated but indeed the test is useless as it doesn't test any actual flow.

If manually doing the steps to reproduce from the initial bugfix PR are not longer an issue, I think it's fine to not return the token.

If we really have time, which we don't, we could replace the dropped test with a test that covers the actual flow that was problematic.

Reply:
The problem is that it sends `access_token=false` instead of leaving it empty.
The original fix should have been to remove the param when false, and not to always send the token.